internal/internal_activity.go

```diff
@@ -51,13 +51,12 @@ type (
 
 	// ActivityID uniquely identifies an activity execution
 	ActivityID struct {
-		scheduleID int64
-		activityID string
+		id string
```

Comment:
It seems like simple
```go
type ActivityID string
```
would be enough here. But current version is extensible. Let's leave it.

Reply:
I considered this, but decided to stick with the struct for now.